tests/test-st2-unauthorized.js

```diff
@@ -0,0 +1,102 @@
+/*
+ Licensed to the StackStorm, Inc ('StackStorm') under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Since this is a new file, new copyright and license headers should be used.

Reply:
Good catch,

updated in https://github.com/StackStorm/hubot-stackstorm/pull/178/commits/8b8504ef9b599c49c7b65dc4aaa144acb230cb49